canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -532,7 +532,8 @@ PropertiesMain.propTypes = {
 		schemaValidation: PropTypes.bool,
 		applyPropertiesWithoutEdit: PropTypes.bool,
 		conditionHiddenPropertyHandling: PropTypes.oneOf(["null", "undefined", "value"]),
-		conditionDisabledPropertyHandling: PropTypes.oneOf(["null", "undefined", "value"])
+		conditionDisabledPropertyHandling: PropTypes.oneOf(["null", "undefined", "value"]),
+		maxLength: PropTypes.number
```

Comment:
I think we might want 2 options.  One for multi-line controls like textarea or expressions and then another for textfield, date, etc... Right now we don't limit those but we might in the future.
We probably also want the ability to set a `-1` for no limit.

Reply:
Adding 2 config options in `propertiesConfig`- 
```
maxLengthForMultiLineControls: PropTypes.number,
maxLengthForSingleLineControls: PropTypes.number
```